plugins/dfu/dfu.quirk

```diff
@@ -441,3 +441,8 @@ RemoveDelay = 60000
 Plugin = dfu
 Flags = detach-for-attach
 RemoveDelay = 180000
+
+# FlatFrog DFU
+[USB\VID_25B5&PID_0004]
+Plugin = dfu
+Flags = manifest-poll,detach-for-attach,ignore-upload
```

Comment:
can you leave a line break at the end please.

Reply:
of course